src/parallel_som.jl

```diff
@@ -167,7 +171,7 @@ function doEpoch(x::Array{Float64}, codes::Array{Float64}, dm::Array{Float64},
      # for each sample in dataset / trainingsset
      for s in 1:nRows
          sampl = vec(x[s, : ])
-         bmu_idx, bmu_vec = find_bmu(codes, sampl)
+         bmu_idx, bmu_vec = findBmu(codes, sampl)
```

Comment:
`idx, vec`

Reply:
`bmuIdx`, `bmuVec` ?

Reply:
so we dont lose track of what it is, because they are used in `doEpoch`